images/linux/ubuntu1804.json

```diff
@@ -126,9 +126,12 @@
         },
         {
             "type": "shell",
-            "inline": [
-                "echo ImageVersion={{user `image_version`}} | tee -a /etc/environment",
-                "echo ImageOS={{user `image_os`}} | tee -a /etc/environment"
+            "scripts": [
+                "{{template_dir}}/scripts/installers/envvariables.sh"
```

Comment:
nitpick: may be `configure-environment.sh` so we will be able to reuse this file in future for some minor system settings

Reply:
Yeah, that was the initial idea. I like your naming better - renamed

Reply:
In that case it makes sense to combine `/updatepath.sh` and `configure-environment.sh` into one script.